### ISSUE TITLE
Probabilistic Boolean Networks

### ISSUE THREAD

**Issue by sgvolpe** [matthiasbock/BooleSim#50]
I wonder if you may have considered to add the option to create Probabilistic Boolean Networks as well.

**Comment by matthiasbock** [comment]
Yes, we have thought about supporting probabilistic networks. They would be quite a powerful expansion and the amount of additional work needed for implementation is relatively limited.

Feel free to make a fork !
I'll be happy to assist with any developments, questions, etc.
